public/index.html

```diff
@@ -48,6 +48,7 @@ <h2>Who's Who at Cohort</h2>
             <li>Jacob Niedzwiecki: Lead Developer</li>
             <li>Nabeel Kassam: Producer/Project Manager</li>
             <li>Luke Garwood: Assistant Developer </li>
+            <li>Amanda Baker: Quality Assurance</li>
```

Comment:
This title works fine, feel free to change it if you like something else better.